src/epub/text/colophon.xhtml

```diff
@@ -18,7 +18,8 @@
 			<a href="https://standardebooks.org">Standard Ebooks</a><br/>
 			by<br/>
 			<b epub:type="z3998:personal-name">Hendrik Kaiber</b>,<br/>
-			and is based on a transcription produced in 2023 by<br/>
+			and is based on a transcription produced in 2024 by<br/>
```

Comment:
Thanks for catching this. The formula is  `This ebook was transcribed and produced for` above, so you don't include his name twice.

Reply:
You're welcome! (I always read every work from Standard Ebooks from cover to cover.)

To fix this, should I edit that file and then do `git push --force`?

Reply:
Actually, I'm a bit a confused. If I change it to `This ebook was transcribed and produced for`, it wouldn't still be possible to include the link to the Wikisource transcription, right? Hendrik first transcribed the novella on Wikisource and then used his transcription from there to produce the Standard Ebooks version.

Reply:
OK, that's a unique case, just revert to the original and put `and is based on a transcription produced in 2023 for<br/>` so that it's implied he also did the transcription. Listing his name twice in a row sounds weird.

Reply:
OK, done! I did it in a new commit. Thanks, Alex.